src/backend/gl/src/queue.rs

```diff
@@ -200,7 +209,7 @@ impl CommandQueue {
     /// Return a reference to a stored data object.
     fn get_raw(data: &[u8], ptr: com::BufferSlice) -> &[u8] {
         assert!(data.len() >= (ptr.offset + ptr.size) as usize);
-        &data[ptr.offset as usize..(ptr.offset + ptr.size) as usize]
+        &data[ptr.offset as usize .. (ptr.offset + ptr.size) as usize]
```

Comment:
Are you using rustfmt nightly? We used that previously, but now we want to switch fully to the stable. That requires cleaning up the rustfmt.toml a bit. The point is - this is an unrelated task, It would be good if your PR didn't attempt to do it partially. Reviewing only non-formatting changes is easier.

Reply:
I normally use stable, but the errors I received hinted that I should use nightly for the gfx. 
This slipped accidentally. Should I just format manually until rustfmt.toml is resolved?

Reply:
yes, please, just don't bother formatting too hard